Sources/NIOTransportServices/NIOTSEventLoopGroup.swift

```diff
@@ -85,4 +85,16 @@ public final class NIOTSEventLoopGroup: EventLoopGroup {
         return EventLoopIterator(self.eventLoops)
     }
 }
+
+@available(OSX 10.14, iOS 12.0, tvOS 12.0, watchOS 6.0, *)
+public struct NIOTSClientTLSProvider: NIOClientTLSProvider {
+    public typealias Bootstrap = NIOTSConnectionBootstrap
+
+    public init() {
```

Comment:
Shouldn't the user be able to specify the `tlsOptions` by providing them here?

Reply:
@glbrntt thank you, yes!